google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -104,6 +107,12 @@ def py(self):
     return Pipeline._imports + default_args + dag_definition + \
         task_definitions + up_steam_statements
 
+  @staticmethod
+  def _get_datetime_expr_str(date_string):
+    expr_format = 'datetime.datetime.strptime(\'{1}\', \'{2}\').replace(tzinfo=timezone(\'UTC\'))'
+    datetime_format = '%Y-%m-%dT%H:%M:%SZ'  # ISO 8601, always UTC
+    return expr_format.format(date_string, date_string, datetime_format)
```

Comment:
You are passing three arguments to a formatting string that only has two replacement fields. I think you can drop the first date_string here, and change the 1 and 2 in the format string to 0 and 1.

Reply:
Good catch, thanks! It's a happy accident that this worked, but thinking again that's probably because I had the {0} there at one point and then removed it.